### ISSUE TITLE
[Action] idle state is not AAA accessible

### ISSUE THREAD

**Issue by SkyeSeitz** [Esri/calcite-components#14148]
**monday.com sync:** #11575038214

### Check existing issues

- [x] I have [checked for existing issues](https://github.com/Esri/calcite-design-system/issues) to avoid duplicates and reviewed the [Accessibility page](https://developers.arcgis.com/calcite-design-system/foundations/accessibility) for guidance.

### Which Component

N/A

### Actual Behavior

Currently the `text.3` text color of Action does not pass a AAA contrast ratio in both light and dark mode. This is also true for any other Calcite components using `text.3`. 

### Expected Behavior

Decide if Action's idle state needs to be adjusted to pass a AAA contrast ratio and if that adjustment should be across the system - possibly at the `text.3` token level. 

A challenge with making an adjustment to the `text.3` token would be maintaining distinction between `text.1`- `text.3` tokens.

<img width="888" height="540" alt="Image" src="https://github.com/user-attachments/assets/d15218c9-8bbd-4032-824f-fd29e61bdbdb" />

See [Figma file for explorations 🔗](https://www.figma.com/design/TT1RHeUb357kVJAPgngdTe/-Action--Dark-mode-idle-state-not-AAA-accessible?node-id=455-4748&t=2vBcK4HGerd27jc2-11)

### Reproduction Sample

https://developers.arcgis.com/calcite-design-system/components/action-bar/

### Reproduction Steps

inspect `text.3` color and check contrast with `surface.1` in [contrast checker](https://webaim.org/resources/contrastchecker/)

### Reproduction Version

5.0

### Priority impact

impact - p3 - not time sensitive

### Calcite package

- [ ] @esri/calcite-components
- [ ] @esri/calcite-components-react
- [ ] @esri/calcite-design-tokens
- [ ] @esri/eslint-plugin-calcite-components

### Esri team

Calcite (design)

**Comment by DintaMel** [comment]
Marked as low priority since our company standard is currently AA.
